cli/cmd/infra_generate.go

```diff
@@ -0,0 +1,41 @@
+package cmd
+
+import (
+	stackservice "github.com/chanzuckerberg/happy/cli/pkg/stack_mgr"
+	"github.com/chanzuckerberg/happy/shared/config"
+	"github.com/chanzuckerberg/happy/shared/util"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	infraCmd.AddCommand(infraGenerateCmd)
+	config.ConfigureCmdWithBootstrapConfig(infraGenerateCmd)
+}
+
+var infraGenerateCmd = &cobra.Command{
+	Use:          "generate",
+	Short:        "Generate HCL code",
+	Long:         "Generate HCL code in environment '{env}'",
```

Comment:
```suggestion
	Short:        "Generate Happy Stack HCL code",
	Long:         "Generate Happy Stack HCL code in environment '{env}'",
```

Make it more clear what this HCL code is